src/types/caldav-parser.d.ts

```diff
@@ -0,0 +1,41 @@
+import { RRule } from 'rrule'
+
+export interface FullCalendarEventObject {
```

Comment:
Why do we have a `FulCalendar` object in a lib which is named `caldav-parser`?

Reply:
the goal is to provide a calendar object that is similar/acceptable by fullcalendar 5.x
https://fullcalendar.io/docs/event-parsing

Reply:
OK yes, but do nit name it `FullCalendarEventObject`, name it `CalendarEventObject`

Reply:
sounds better, thank you